specification/azsadmin/resource-manager/commerce/readme.md

```diff
@@ -61,27 +61,11 @@ csharp:
   clear-output-folder: true
 ```
 
-## Multi-API/Profile support for AutoRest v3 generators 
+# AutoRest v3 generators
 
-AutoRest V3 generators require the use of `--tag=all-api-versions` to select api files.
-
-This block is updated by an automatic script. Edits may be lost!
-
-``` yaml $(tag) == 'all-api-versions' /* autogenerated */
-# include the azure profile definitions from the standard location
-require: $(this-folder)/../../../../profiles/readme.md
+> see https://aka.ms/autorest
 
-# all the input files across all versions
+``` yaml
 input-file:
-  - $(this-folder)/Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json
-
+  - Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json
 ```
```

Comment:
I am not quite sure why this is deleted. This segment should be auto-generated everytime when a PR gets merged. The changes here may be overridden